### DISCUSSION TITLE
Inconsistent follower count between mobile and website

### DISCUSSION THREAD

**Discussion by hanxiaop** [community/community#118161] [Mobile]
### Select Topic Area

Bug

### Body

The mobile app shows I have 35 followers but the website shows I have 36 😂.

mobile:
<img width="304" alt="image" src="https://github.com/community/community/assets/20888172/acd98dea-94cd-49e8-ba39-99213f03e7d5">

web:
<img width="249" alt="image" src="https://github.com/community/community/assets/20888172/5ffae87d-46a5-4575-8344-8b1a741ba244">

**Comment by warioddly** [comment]
Maybe the mobile version is caching, I just refresh the page once or more until I see the result (looks like a bug)
